src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends FigureAbstract implements Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea() + ", radius: " + radius + ", color: " + getColor());
```

Comment:
There is a typo in the `draw` method's output string. It should be 'area' instead of 'are'.